src/index.js

```diff
@@ -1,54 +1,73 @@
 'use strict';
+// В якості покращення роботи бібліотеки "tags-input запропоновано
```

Comment:
good idea!
but for the future please use "English" for descriptions
and one more never put comments which described how the code works!

Reply:
👍